test/plugin_configuration_test.rb

```diff
@@ -66,7 +70,7 @@ def test_provider_configuration_with_host_crud
     assert_not_nil provider.free_ips
     assert_equal @network_view, provider.network_view
     assert provider.managed_subnets.include?('1.1.1.0/255.255.255.0')
-    assert provider.crud.instance_of?(::Proxy::DHCP::Infoblox::HostIpv4AddressCRUD)
+    assert provider.crud.instance_of?(::Proxy::DHCP::Infoblox::FixedAddressCRUD)
```

Comment:
Previous commit changed default setting to "fixedaddress" instead of host and I missed Travis, this fixes tests.